aict-MERN/backend-of-aict-site-w-react/db.js

```diff
@@ -0,0 +1,25 @@
+// require('dotenv').config()
+import dotenv from 'dotenv'
+dotenv.config()
+
+import pkg from 'pg';
+const {Pool} = pkg;
+const pool = new Pool({
+  user: process.env.PSQLUSER,
+  host: process.env.PSQLHOST,
+  database: 'aictforum',
+  password: null,
+  port: 5432,
+})
+
+
+
+export const getPosts = async function() {
+  let data = await pool.query('SELECT * FROM posts')
+  return data.rows.reverse()
+}
+export const postPost = async function(values) {
+  const text = `INSERT INTO posts(posted_by, post, date_of_post) VALUES($1, $2, $3)`
+  await pool.query(text, values)
+}
+export default pool
```

Comment:
this whole file is exactly what i wanted you to do in my review for the last PR. well done!

EDIT: with one small exception.. x)

first, real quick, you can rewrite the following
```js
export const postPost = async function(values) { ...
````
like this:
```js
export async function postPost(values) { ...
```

second, for the postPost functions, its more intuitive for the function to expect a post object when you call it. hence, the signature should look something like this:
```js
function postPost(post) { ...
```
and the array that you pass to `pool.query` would be constructed in-place something like `[post.poster, post.text, post.date]` (following the same logic you use in the `routes/posts.js` file

Reply:
That is so much cleaner, thank you for pointing this out, its really handy for the export to just be one big obj